### ISSUE TITLE
Error Occurred: The number of results does not match the number of interviews that were executed in a single bulk execution request.

### ISSUE THREAD

**Issue by Staples1887** [paull10au/qsyd_InvocableEmailActions#9]
## Summary
Error element Send_Template (FlowActionCall).
The number of results does not match the number of interviews that were executed in a single bulk execution request.

## Steps To Reproduce

1. Create flow using invocable action QSYD_INVOCABLEEMAILTEMPLATEACTION
2. Run the flow using a schedule flow 
3. Get error in an email

![Send Email error](https://user-images.githubusercontent.com/59692325/106263107-dc06bc00-621b-11eb-9f23-a68a7e441375.png)

## Expected Behavior
Emails sent to each record

## Actual Result
Error element Send_Template (FlowActionCall).
The number of results does not match the number of interviews that were executed in a single bulk execution request.

## Device? 
- Salesforce Flow

**Comment by paull10au** [comment]
Thanks @Staples1887,

I have updated to return a list of List<qsyd_InvocableEmailResult> to match the number of input parameters as per 

https://salesforce.stackexchange.com/questions/321274/bulkification-of-flow-with-invocable-action-error-re-number-of-results-how

However, when testing your use case of trying to send emails via scheduled flows, I noticed a few things.

1. The Automated user needed permission to execute the invocable email classes as per https://salesforce.stackexchange.com/questions/210710/automated-process-user-profile-permissions
2. The Automated user lacks permissions to execute Messaging.SendEmail as per https://salesforce.stackexchange.com/questions/231642/for-the-platform-event-automated-process-user-why-do-both-visualforce-control

I have push my changes to the following branch The-number-of-results-does-not-match-the-number-of-interviews-that-were-executed-in-a-single-bulk-execution-request for you to have a look at.

let me know if this resolves your error and if you get emails to send.

A quick read of workarounds are using an external email service, cyclic org reference via connected app.
